src/app/dashboard/under-notice/under-notice.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="nhsuk-table-responsive" *ngIf="trainees$ | async as trainees">
+  <table class="nhsuk-table">
+    <caption class="nhsuk-table__caption">
+      Trainees under notice
+    </caption>
+    <thead class="nhsuk-table__head">
+      <tr class="nhsuk-table__row">
+        <th class="nhsuk-table__header" scope="col">Date added</th>
+        <th class="nhsuk-table__header" scope="col">First name</th>
+        <th class="nhsuk-table__header" scope="col">Last name</th>
+        <th class="nhsuk-table__header" scope="col">Gmc Reference no</th>
```

Comment:
Naming convention?

Reply:
the <table> markup/css classes have been leveraged from `https://service-manual.nhs.uk/design-system/components/table`